config/prow/cluster/monitoring/mixins/prometheus/ci_absent_alerts.libsonnet

```diff
@@ -19,16 +18,7 @@
               message: '@test-infra-oncall The service %s has been down for 5 minutes.' % name,
             },
           }
-          for name in [
-              comps.crier,
-              comps.deck,
-              comps.ghproxy,
-              comps.hook,
-              comps.horologium,
-              comps.prowControllerManager,
-              comps.sinker,
-              comps.tide,
-          ]
+          for name in $._config.ciAbsents.components
```

Comment:
nit: This is not backwards compatible without defaulting. Not sure if we care yet given that updates are currently manual anyways.

Reply:
ack, will be careful when updating other prow monitoring stacks